### DISCUSSION TITLE
Add Boundary Conditions in output of feelpp_toolboxes exec

### DISCUSSION THREAD

**Discussion by Trophime** [feelpp/feelpp#1637] [Ideas]
It will be helpful to have the list of boundary conditions printed in the output of feelpp_toolboxes exec
just like we have the materials.

This feature seems to have disappeared from latest version.

**Comment by vincentchabannes** [comment]
Boundary conditions are not printed because the output system has changed. It requires adding some codes but I planned to review fully the base class of boundary conditions (and therefore not to do the work twice). It will be available as soon as possible...
